Assets/Scripts/Combat/PlayerController.cs

```diff
@@ -131,5 +133,12 @@ private void CheckMouseClick()
                 SetMouseHover();
             }
         }
+        if (Input.GetKeyDown("r"))
```

Comment:
I would use GetButtonDown instead of GetKeyDown

Reply:
We'll eventually want to replace this with a UI button, I think, so I'd rather leave it as a key for now than reconfigure input settings.